### DISCUSSION TITLE
can i send https request use $fetch in server api endpoint?

### DISCUSSION THREAD

**Discussion by TangsWang** [nuxt/nuxt#16952] [Questions]
### Describe the feature

now, there's a problem `unable to verify the first certificate`, cuz https。

can i make some config to let it ok?

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [ ] Read the [contribution guide](https://v3.nuxtjs.org/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/framework/discussions) and [issues](https://github.com/nuxt/framework/issues).

**Comment by palamarchukser** [comment]
[Using rejectUnauthorized](https://stackoverflow.com/questions/60061143/using-rejectunauthorized-with-node-fetch-in-node-js) - it's ok for you?

**Reply by TangsWang** [reply]
thanks, it do not work for me.  cuz github do not allow unauthorized.

but i found `ssl-root-cas`,  it works for me  and nuxt3's $fetch.
